android/src/com/mapswithme/maps/uber/Uber.java

```diff
@@ -10,8 +10,8 @@ public static native void nativeRequestUberProducts(@NonNull NetworkPolicy polic
                                                       double srcLon, double dstLat, double dstLon);
 
   @NonNull
-  public static native UberLinks nativeGetUberLinks(@NonNull String productId, double srcLon, double srcLat,
-                                                    double dstLat, double dstLon);
+  public static native UberLinks nativeGetUberLinks(@NonNull NetworkPolicy policy,
```

Comment:
I would do it in a separate PR or in scope of this task https://jira.mail.ru/browse/MAPSME-4615 if you don't insist. In any case, I think all renaming changes should be in a separate commit.